doc/api/net.md

```diff
@@ -114,7 +114,7 @@ Emitted when the server has been bound after calling [`server.listen()`][].
 added: v0.1.90
 -->
 
-* Returns: {Object}
+* Returns: {Object} | {string}
```

Comment:
Needs to be `{Object|string}`.  Otherwise the generated JSON looks like this:

```json
                  "return": {
                    "textRaw": "Returns: {Object} | {string}",
                    "name": "return",
                    "type": "Object",
                    "desc": "| {string}"
                  },
```